WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentView.swift

```diff
@@ -60,7 +63,11 @@ struct BlazeConfirmPaymentView: View {
                         .padding(.horizontal, Layout.contentPadding)
                 }
 
-                Divider()
+                if sizeCategory.isAccessibilityCategory {
+                    footerView
```

Comment:
Nit: Maybe a spacer above the footer view would push the content down when there is remaining vertical space?

```suggestion
                    Spacer()
                    footerView
```

<img src="https://github.com/user-attachments/assets/8e47dbad-379c-4bf4-98b4-56d8d582eb94" width=320 />